include/deal.II/grid/grid_in.h

```diff
@@ -577,7 +620,34 @@ class GridIn
   Format default_format;
 };
 
-
+namespace internal
+{
+  template <int dim>
+  class Abaqus_to_UCD
+    {
+    public:
+      Abaqus_to_UCD (void);
+      ~Abaqus_to_UCD (void);
+
+      bool read_in_abaqus (std::istream &in);
+      bool write_out_avs_ucd (std::iostream &out);
```

Comment:
I was wondering... should the *Abaqus_to_UCD* class be further embedded in another namespace, i.e. internal::Grid_In::Abaqus_to_UCD ?